### PR TITLE
added several examples

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/300

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @e0i, for your information

Thank you

File: docset/windows/hyper-v/New-VM.md [hunk 1]
```diff
@@ -83,6 +83,33 @@ New-VM -Name "new 3" -MemoryStartupBytes 1GB -VHDPath d:\vhd\BaseImage.vhdx
...
+
+This example asks for credentials, then creates a virtual machine named new 4 that has 2 GB of memory on the server named HostServer01.
+
+### Example 5
+
+```powershell
+New-VM -Name "new 5" -Generation 1 -BootDevice CD -NoVHD
+```
+
+This example creates a virtual machine named new 5. The machine doesn't have any VHD disk and set to boot from CD.
```
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/New-VM.md:100]:
```suggestion
This example creates a virtual machine named new 5. The machine doesn't have any VHD disk and is set to boot from CD. 
```

File: docset/windows/hyper-v/New-VM.md [hunk 2]
```diff
@@ -83,6 +83,33 @@ New-VM -Name "new 3" -MemoryStartupBytes 1GB -VHDPath d:\vhd\BaseImage.vhdx
...
+### Example 6
+
+```powershell
+$oldVM = Get-VM "old 1"
+$memory = (Get-VMMemory -VMName $oldVM.name).Startup
+$switch = (Get-VMNetworkAdapter -VMName $oldVM.name).SwitchName
+New-VM -Name "new 6" -Generation $oldVM.Generation -MemoryStartupBytes $memory -SwitchName $switch
+```
+
+This example creates a virtual machine named new 6. The machine has the same generation and amount of assigned memory as existing machine named old 1, and connected to the same network switch.
```
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/New-VM.md:111]:
```suggestion
This example creates a virtual machine named new 6. The machine has the same generation and amount of assigned memory as the existing machine named old 1 and connects to the same network switch.
```

File: docset/windows/hyper-v/New-VM.md [hunk 3]
```diff
@@ -83,6 +83,33 @@ New-VM -Name "new 3" -MemoryStartupBytes 1GB -VHDPath d:\vhd\BaseImage.vhdx
 
 This example creates a virtual machine named new 3 that has 1 GB of memory and connects it to an existing virtual hard disk that uses the VHDX format.
 
+### Example 4
+
+```powershell
+New-VM -Name "new 4" -MemoryStartupBytes 2GB -Credential (Get-Credential) -ComputerName HostServer01
+```
+
+This example asks for credentials, then creates a virtual machine named new 4 that has 2 GB of memory on the server named HostServer01.
```
Comment (JohanFreelancer9) [inline docset/windows/hyper-v/New-VM.md:92]:
```suggestion
This example asks for credentials, then creates a virtual machine named new 4, which has 2 GB of memory, on the server named HostServer01.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Review by dariomws — APPROVED** [review]
These updates are good for me.

Anyway, I think the author of the issues is also asking us to fill in the missing descriptions of the parameters. And, we can add/update the same information for the W2012 and W2012 R2 versions.

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by MaratMussabekov** [comment]
hello, added for WS2012 and WS2012 R2
Thank you

**Pull request merged**